tinygrad/ops.py

```diff
@@ -545,7 +547,6 @@ def flops_mem(uops:List[UOp], ignore_indexing=False) -> Tuple[sint, sint]:
         if len(u.src) > 3: dont_count = dont_count.union(u.src[2].sparents)
       elif u.op is UOps.STORE:
         dont_count = dont_count.union(u.src[1].sparents)
-        if len(u.src) > 3: dont_count = dont_count.union(u.src[3].sparents)
```

Comment:
This was causing op_estimate to be 0.

And then it would be set as the smallest compute op during beam search. And then any following beam searches would be skipped because they would have compute ops > 0. (https://github.com/tinygrad/tinygrad/blob/master/tinygrad/engine/search.py#L157)